datajoint/autopopulate.py

```diff
@@ -118,12 +122,12 @@ def populate(self, *restrictions, suppress_errors=False, reserve_jobs=False, ord
                             jobs.complete(self.target.table_name, key)
         return error_list
 
-    def progress(self, restriction=None, display=True):
+    def progress(self, *restrictions, display=True):
```

Comment:
it's interesting but not used anywhere else yet.  Yes, we could add this syntax.  The `progress` should not use any other arguments such as `display` or may be prefix it as `_display` to avoid name collisions.